portfolio/src/main/webapp/comment.html

```diff
@@ -5,16 +5,26 @@
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="https://kit.fontawesome.com/e073fdea10.js" crossorigin="anonymous"></script>
-    <script src="script.js"></script>
+    <script src="comment-form.js"></script>
   </head>
-  <body class="container comment-container">
+  <body class="container comment-container" onload="getComments()">
```

Comment:
No action required

FYI you can also attach this via javascript

```javascript
document.on('DOMContentLoaded', getComments);
```

As your pages get more complex, it would be easier to pick one consistent way to attach event listeners. I think the JavaScript method is a bit more flexible, as you can attach listeners to dynamically created elements, but many frameworks follow the markup style (Angular and React commonly have you attach listeners in their templates). Ultimately, you want consistent code so it's easier to maintain and add new features to.

Reply:
This makes sense! I will definitely keep this in mind going forward